ts/src/bitget.ts

```diff
@@ -1464,9 +1464,9 @@ export default class bitget extends Exchange {
                 'networks': networks,
                 'type': undefined,
                 'name': undefined,
-                'active': deposit && withdraw,
-                'deposit': deposit,
-                'withdraw': withdraw,
+                'active': isTokenDepositable && isTokenWithdrawable,
```

Comment:
Same question, why do we disable it while we may be able to trade tokens we already have inside the CEX?

I mean can it be tradable but not depositable and withdrawable? So should we disable it?

Reply:
We don't use this property at all, because it's not clear whether CCXT defines it only for deposits and withdrawals, or also for trading